modules/ROOT/pages/migration-dataweave.adoc

```diff
@@ -18,10 +18,12 @@ endif::[]
 
 Most of the header directives have been changed in DataWeave. The `%dw` directive is an exception. In most cases, MuleSoft removed the `%`, and in some cases the keyword is shortened.
 
-[%header,cols=“1,1”]
+[%header%autowidth.spread,cols="a,a,a"]
 |===
-|DataWeave 1.0 |DataWeave 2.0 |
+|DataWeave 1.0 |DataWeave 2.0 | Comments
 |`%dw 1.0`    |`%dw 2.0`|
+|`%input`     |`input`  | In a Mule app flow, DataWeave 2.0 recognizes the input type based on the `output` directive specified in the preceding component. The input type is not defined using the `input` directive in the header anymore.
```

Comment:
Perhaps TMI, but you can also identify (not set or change) an input MIME type via a DW `contentType` param to read and readUrl methods. https://docs.mulesoft.com/mule-runtime/4.3/dw-core-functions-read